### ISSUE TITLE
How much data need for pre-training the language models from pre-trained checkpoints

### ISSUE THREAD

**Issue by ankush20m** [google-research/bert#991]
Hi, 

- What's the primary difference between pre-training from pre-trained checkpoints and fine-tuning the models?

- How much data do I need to pre-train the model?

- What will be the format of the custom corpus, will it be a **text file** containing sequences per line?

- Could you suggest any other way how can I fine-tune the BERT with the unsupervised dataset (no label dataset, an only text file containing sequences)?
